### PR TITLE
Remove migration code and info

### PR DESCRIPTION
**Pull request by Baggerone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,6 @@
 # Terraform Enterprise Migration Tool
```
Comment (Schparky) [inline README.md:1]:
```suggestion
# Terraform Ops Tool
```

File: README.md [hunk 2]
```diff
@@ -59,95 +50,36 @@ List the workspaces with at least one of their attributes/pieces of data.
...
-
-## Example
-Here is a video of us using this script to migrate a dozen environments from legacy to new. You can see the script 
-running on the left and the Terraform Enterprise interface on the right to see the new workspaces just appearing as the 
-script runs. You can see how much time this script saves when we move 12 environments in 3 minutes. Manually it would 
-have taken closer to 3 hours!
-
-![Example video](video.gif)
-
 ## Usage
```
Comment (Schparky) [inline README.md:53]: Looks like you could remove `video.gif`

File: README.md [hunk 3]
```diff
@@ -59,95 +50,36 @@ List the workspaces with at least one of their attributes/pieces of data.
...
 
 ### General Help
 ```text
-$ terraform-enterprise-migrator -h
-Migration is a three step process. First run plan, modify the generated file as needed,
-then run migrate to process the plan file
+$ terraform-cloud-ops -h
+Perform TF Cloud operations, e.g. clone a workspace or search for variables within a workspace
```
Comment (Schparky) [inline README.md:58]: There are a few instances of "search for variables" that could be changed to something like "manage variables" to be more inclusive of all of the current (and future) features.

**Review by Schparky — APPROVED** [review]
Looks good, just a few comments here and there.

**Review by fillup — APPROVED** [review]

**Pull request merged**